DashWallet/Sources/Models/CrowdNode/Services/SendCoinsService.swift

```diff
@@ -57,7 +57,7 @@ public final class SendCoinsService {
             }
         }
 
-        await account.sign(transaction, withPrompt: nil)
+        await account.sign(transaction)
```

Comment:
For the signup and withdrawals, we always authenticate manually in the viewModel, so no need to use different versions of this method here.